esp/services/ws_loggingservice/loggingserviceplugin.cpp

```diff
@@ -42,7 +42,7 @@ extern "C"
     ESP_FACTORY IEspRpcBinding * esp_binding_factory(const char *name, const char* type, IPropertyTree *cfg, const char *process)
     {
         if (streq(type, "loggingservice_binding"))
-            return new CWsLoggingServiceSoapBinding(cfg, name, process, hsl_none);
+            return new CWsLoggingServiceSoapBindingEx(cfg, name, process, hsl_none);
```

Comment:
Not sure what this bought you, but OK.

Reply:
In one of my many iterations of this change I believe I was implementing/overriding some of the new detach/attach subscribe/unsubscribe methods, not needed now.